ts/store/reducers/navigation.ts

```diff
@@ -21,6 +22,9 @@ export const navigationStateSelector = (state: GlobalState): NavigationState =>
 
 function nextState(state: NavigationState, action: Action): NavigationState {
   switch (action.type) {
+    // Used to restart application after logout
```

Comment:
This bug is deeper as expected.
A first step that solves the issue could be:

- revert changes you applied
- edit _ts/sagas/startup/watchLogoutSaga.ts_ by removing `yield put(sessionExpired());` where it is and add a **finally** block and put it inside. **sessionExpired** will be dispatched anyway

Check it out if this solution could work